Open Judge System/Services/OJS.Services/Business/ParticipantScores/ParticipantScoresBusinessService.cs

```diff
@@ -1,4 +1,13 @@
-﻿namespace OJS.Services.Business.ParticipantScores
+﻿using System;
+using System.Collections.Generic;
```

Comment:
move usings in a namespace or move the usings below which are in the namespace of it to be consistent